rootsrc/init

```diff
@@ -32,6 +32,11 @@ done </etc/modules
 
 (netd &)
 
+# try to start sshd if the sshd layer was mounted by initrd
+if which startsshd &>/dev/null; then
```

Comment:
I don't think &> works in dash.  Have to use the >/dev/null 2>&1 business.

Reply:
good catch. fixed in 5c4ea4a.